frontend/tests/utils/xgeneUtils.ts

```diff
@@ -0,0 +1,113 @@
+import { ROUTES } from "src/common/constants/routes";
+import { TEST_URL } from "../common/constants";
+import { expect, Page } from "@playwright/test";
+import { getTestID, getText } from "tests/utils/selectors";
+
+export function goToWMG(page: Page) {
+  return Promise.all([
+    page.waitForResponse(
+      (resp: { url: () => string | string[]; status: () => number }) =>
+        resp.url().includes("/wmg/v1/filters") && resp.status() === 200
+    ),
+    page.goto(`${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`),
+  ]);
+}
+
+export const selectFIlterOption = async (page: Page, filterName: string) => {
+  const FIRST_OPTION = '[data-option-index="0"]';
+  // click the filter at the corner this is done due to the fact that the default click is being intercepted by another element
```

Comment:
Can we remove the space between this comment and the actual code please? Didn't realize the comment what was the code below. Thanks for explaining!

Reply:
What element is intercepting the click on the option? We have other tests that select filter options don't seem to run into this issue!